docker/README.md

```diff
@@ -90,10 +90,21 @@ You need to install [nvidia-docker plugin](https://github.com/NVIDIA/nvidia-dock
 Once you install `nvidia-docker`, run following commands (for info modifying the tag, see the CPU section above):
 
 ```bash
-cp Dockerfile.gpu Dockerfile
-docker build -t mms_image_gpu .
+docker build -f Dockerfile.gpu -t mms_image_gpu .
 ```
 
+## Preparing `models` and `mms_app.conf` files for running the Model Server
+Create a `models` directory on the host machine and add the models to be used along with the mms_app_[cpu/gpu].conf file into the directory. Modify the `mms_app[cpu/gpu].conf` file to reflect the model files to be used along with updated options for other `gunicorn`, `nginx` and `MMS` configurations.
+ ```bash
```

Comment:
this script is a bit weird... 
- the pwd and ls commands/output should go away. 
- the cp command refers to home directory path that may match anything on reader's system
- the comment aobut modifying conf is in an odd spot

maybe 
\# modfy the conf.... file
mkdir models
cp mms_app_cpu.conf models/

Reply:
Ya, I was trying to be descriptive. Probably ended being more confusing. Changed it.